mantidproject/mslice
# Plot interactivity - slices

Features of a slice plot can now be changed by clicking on them (issue #180). This includes Labels, legends, axis ranges, and overplot lines. This functionality will be extended to cuts in the future.

**To test:**
-  Plot a slice
- Click a feature (axis labels or ranges, legends, overplot lines)
- Change settings in the dialog
- Click OK
- Check that the plot is updated with new settings
- Repeat for each feature

@martyngigg
